src/taco-remote-lib/ios/iosEmulateHelper.ts

```diff
@@ -32,17 +32,19 @@ process.on("message", function (emulateRequest: { appDir: string; appName: strin
 
 class IOSEmulateHelper {
     private static IOS_SIMULATOR_TARGETS: { [id: string]: string } = {
-        "iphone 4s": "--retina",
-        "iphone 5": "--retina --tall",
-        "iphone 5s": "--retina --tall --64bit",
-        "iphone 6": "--devicetypeid com.apple.CoreSimulator.SimDeviceType.iPhone-6",
-        "iphone 6 plus": "--devicetypeid com.apple.CoreSimulator.SimDeviceType.iPhone-6-Plus",
-        "ipad 2": "--family ipad",
-        "ipad air": "--family ipad --retina --64bit",
-        "ipad retina": "--family ipad --retina"
+        "iphone 4s": "iPhone-4s",
```

Comment:
Yeah, ios-sim 4 (and 5) changed how simulator targets are specified. You now specify `--devicetypeid 'simulatorTarget, iosVersion'` e.g. `--devicetypeid 'iPad-2, 9.0'`